connector_helpscout/models/helpscout_user/importer.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+import urllib2
+
+from odoo.addons.component.core import Component
+from odoo.addons.connector.components.mapper import mapping, none
+
+_logger = logging.getLogger(__name__)
+
+
+class HelpScoutUserImportMapper(Component):
+    _name = 'helpscout.import.mapper.user'
+    _inherit = 'helpscout.import.mapper'
+    _apply_on = 'helpscout.user'
+
+    direct = [(none('first_name'), 'firstname'),
+              (none('last_name'), 'lastname'),
+              (none('email'), 'login'),
+              (none('email'), 'email'),
+              (none('timezone'), 'tz'),
+              (none('type'), 'helpscout_type'),
+              ('created_at', 'backend_date_created'),
+              ('modified_at', 'backend_date_modified'),
+              ]
+
```

Comment:
We need to map to pre-existing users here.

```python
@mapping
@only_create
def odoo_id(self, record):
    user = self.env['res.users'].search([('email, '=', record.email)])
    if user:
        return {'odoo_id': user.id}
```

Please add same logic to customers. Might make sense to inherit one from the other, but I can't remember off hand

Reply:
Would it be better to search odoo `login`, rather than `email`? Login is a required field, email is not.

Reply:
Only applies to `res.users` though, `res.partner` doesn't have it.

Reply:
Nope because login is not the same in some instances. Add a note to the readme that mentions the user emails are assumed to be correct

Also maybe add a filter to the domain for only internal users, which would disallow matching customers (and reverse that for customer)

Reply:
(think about your Active Directory login - our customers use similar authentication approaches)

Reply:
I would be OK with an option in the backend on whether to match the user by email or login as well

Reply:
I added the filter to not match imported customers to internal users, but I believe it's unnecessary for imported users if we're already searching on the `res.users` model, correct?

Reply:
Customers can have user accounts (e-commerce). You delineate using the `share` attribute on users. If it's `True`, it's a customer. `False` is employee